apps/daffio/.eslintrc.js

```diff
@@ -16,6 +16,9 @@ module.exports = {
         createDefaultProgram: true
       },
       rules: {
+        '@angular-eslint/prefer-on-push-component-change-detection': [
+          'error'
+        ],
```

Comment:
Do you think this should be part of our published eslint config? I think so.

Reply:
Yes, absolutely. But, we have large number of components in the repo that need to be checked before we have support across Daffodil and therefore are able to add it to our global config.

Reply:
@xelaint can you add a todo to remove this once this is part of the global config?

Reply:
Added